apps/android/app/src/main/java/chat/simplex/app/model/ChatModel.kt

```diff
@@ -211,27 +211,34 @@ class ChatModel(val controller: ChatController) {
     }
   }
 
-  fun markChatItemsRead(cInfo: ChatInfo) {
-    // update preview
```

Comment:
please revert the order change so I can see the diff

Reply:
I didn't do it just because I like that order. 

https://github.com/simplex-chat/simplex-chat/pull/925/files#diff-680099d115984a292f562ee96bf0e78965557165b4dd49b2aa20c80383ad33aeR216

This line shows that I need to calculate `markedRead` first before using this variable here: https://github.com/simplex-chat/simplex-chat/pull/925/files#diff-680099d115984a292f562ee96bf0e78965557165b4dd49b2aa20c80383ad33aeR236

Since I need to calculate how many I marked as read. Now there is possibility to mark individual items as read not just the whole chat.